bokeh/models/tiles.py

```diff
@@ -26,11 +26,11 @@ class TileSource(PlotObject):
     the maximum zoom level for the tile layer. This is the most "zoomed-in" level.
     """)
 
-    x_origin_offset = Float(default=20037508.34, help="""
+    x_origin_offset = Float(default=None, help="""
```

Comment:
That warning is from properties.py because the property system doesn't really support "abstract properties" in the base class. Here I think I'd remove those three properties from the TileSource base class.